webapp/packages/core-ui/src/Form/FormBaseService.ts

```diff
@@ -20,12 +20,11 @@ export class FormBaseService<TState, TProps extends IFormProps<TState> = IFormPr
   readonly parts: TabsContainer<TProps>;
   readonly actionsContainer: PlaceholderContainer<TProps>;
 
-  readonly onConfigure: IExecutorHandlersCollection<IFormState<TState>>;
-  readonly onFillDefaultConfig: IExecutorHandlersCollection<IFormState<TState>>;
   readonly onPrepareConfig: IExecutorHandlersCollection<TState>;
```

Comment:
seems like we don't use `onPrepareConfig` also

Reply:
please also check other executors

Reply:
This was the last one